src/styles/bookList.css

```diff
@@ -0,0 +1,20 @@
+main {
+  position: absolute;
+  width: 100%;
+  z-index: -1;
+  top: 115px;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
- [ ] Finally, let's take a look at the general design. We need to do some adjustments in terms alignment of different elements on the page. Take a look at the screenshots below to know areas that need adjustments.

| The Provided Design | Your implementation |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/45002654/150172344-28e29268-00b5-4440-a5a1-c797b14d3d75.png) | ![image](https://user-images.githubusercontent.com/45002654/150168372-92901922-108b-49c9-b708-e4c0617df6aa.png) |

Looking at the two red lines on the screenshot to the right, they get to show how we have not really aligned the given elements properly on the page. Please take care of margins, padding, and widths of components/elements that seem out of place.

Reply:
done